### PR TITLE
Release v0.11.5

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Darwinia Try Runtime
```
2021-11-01 16:22:16 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0
2021-11-01 16:22:16 [#0] 🗳  Finalized election round with compute ElectionCompute::OnChain.
2021-11-01 16:22:16 [0] 💸 new validator set of size 1 has been processed for era 1
2021-11-01 16:22:16 initializing remote client to "wss://rpc.darwinia.network"
2021-11-01 16:22:16 scraping key-pairs from remote @ 0x9d4e36becf8ffb5f2454a753d41d10eff36a5fcd0c8de8a9ae9a56e4c704aa8b
2021-11-01 16:22:16 downloading data for all modules.
2021-11-01 16:23:17 Querying a total of 126216 keys
2021-11-01 16:24:14 extending externalities with 1 manually injected keys
2021-11-01 16:24:14 injecting a total of 126217 keys
2021-11-01 16:24:17 try-runtime::on_runtime_upgrade Darwinia.
2021-11-01 16:24:17 pre-migration grandpa test with new = Grandpa
2021-11-01 16:24:17 Running migration to v3.1 for grandpa with storage version Some(<wasm:stripped>)
2021-11-01 16:24:17 new prefix: Grandpa
2021-11-01 16:24:17 ⚠️ System declares internal migrations (which *might* execute), setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for TronBacking, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for EthereumRelayAuthorities, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for EthereumRelayerGame, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for EthereumBacking, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for EthereumRelay, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Multisig, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Proxy, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Scheduler, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Recovery, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Society, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Identity, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Utility, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Vesting, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Sudo, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Bounties, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Tips, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for KtonTreasury, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Treasury, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for TechnicalMembership, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for PhragmenElection, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for TechnicalCommittee, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Council, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Democracy, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for DarwiniaHeaderMMR, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for AuthorityDiscovery, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for ImOnline, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Grandpa, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Session, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Historical, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ⚠️ Offences declares internal migrations (which *might* execute), setting storage version to <wasm:stripped>
2021-11-01 16:24:17 have 0 deferred offences, applying.
2021-11-01 16:24:17 ✅ no migration for Staking, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for ElectionProviderMultiPhase, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Authorship, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for TransactionPayment, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Kton, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Balances, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Timestamp, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for Babe, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 ✅ no migration for RandomnessCollectiveFlip, setting storage version to <wasm:stripped>
2021-11-01 16:24:17 post-migration grandpa
2021-11-01 16:24:17 TryRuntime_on_runtime_upgrade executed without errors. Consumed weight = 513056000000000, total weight = 512000000000000 (1.0020625)
```

Crab Try Runtime
```

```

File: runtime/crab/src/pallets/evm_.rs [hunk 1]
```diff
@@ -52,16 +53,23 @@ impl<R: darwinia_evm::Config> PrecompileSet for CrabPrecompiles<R> {
 	}
 }
 
+pub struct FixedGasPrice;
+impl FeeCalculator for FixedGasPrice {
+	fn min_gas_price() -> U256 {
+		U256::from(1 * COIN)
```
Comment (HackFisher) [inline runtime/crab/src/pallets/evm_.rs:59]: @AsceticBear @hujw77  will 1 coin be too large or small?

File: runtime/darwinia/src/lib.rs [hunk 2]
```diff
@@ -590,28 +592,38 @@ sp_api::impl_runtime_apis! {
...
-	// TODO: Move to S2S
-	// const CrabBackingPalletId: PalletId = PalletId(*b"da/crabk");
+		// TODO: Move to S2S
+		// const CrabBackingPalletId: PalletId = PalletId(*b"da/crabk");
 
-	0
-	// RuntimeBlockWeights::get().max_block
-}
+		let name = <Runtime as frame_system::Config>::PalletInfo::name::<Grandpa>()
+			.expect("grandpa is part of pallets in construct_runtime, so it has a name; qed");
```
Comment (HackFisher) [inline runtime/darwinia/src/lib.rs:605]: will the expect here be dangerous?

Comment (AurevoirXavier) [reply runtime/darwinia/src/lib.rs:605]: https://github.com/paritytech/polkadot/pull/2994#discussion_r730644771

File: runtime/crab/src/pallets/evm_.rs (hunk 1 above)
Comment (hujw77) [reply runtime/crab/src/pallets/evm_.rs:59]: `1 COIN = 1e9` it's right?

File: runtime/darwinia/src/lib.rs (hunk 2 above)
Comment (HackFisher) [reply runtime/darwinia/src/lib.rs:605]: > https://github.com/paritytech/polkadot/pull/2994#discussion_r730644771

I agree your opinion on this, both are fine, but if let Some is better than expect.

**Comment by HackFisher** [comment]
cc @sxlwar @freehere107 for the type and substrate deps changes.

File: runtime/crab/src/pallets/evm_.rs (hunk 1 above)
Comment (HackFisher) [reply runtime/crab/src/pallets/evm_.rs:59]: yes，but coin here is kind of miss leading? it is 1gwei and 10^-9 RING cause in evm the decimal is 10^18? how may gas for transfer ether? 21000？

Comment (AsceticBear) [reply runtime/crab/src/pallets/evm_.rs:59]: See the screenshot here https://github.com/darwinia-network/darwinia/pull/765#issue-1037402454

Comment (HackFisher) [reply runtime/crab/src/pallets/evm_.rs:59]: @AurevoirXavier Please change to 10 GWEI, and replace "COING" with "GWEI".

GWEI = 10^9

**Review by AsceticBear — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
@HackFisher Any other questions?

**Review by HackFisher — APPROVED** [review]

**Pull request merged**